turbopack/crates/turbopack-ecmascript/src/simple_tree_shake/mod.rs

```diff
@@ -0,0 +1,101 @@
+//! Intermediate tree shaking that uses global information but not good as the full tree shaking.
+
+use anyhow::{Context, Result};
+use rustc_hash::FxHashMap;
+use turbo_rcstr::RcStr;
+use turbo_tasks::{ResolvedVc, Vc};
+use turbopack_core::{
+    module_graph::{ModuleGraph, SingleModuleGraph},
+    resolve::Export,
+};
+
+use crate::chunk::EcmascriptChunkPlaceable;
+
+#[turbo_tasks::function]
+pub async fn is_export_used(
+    graph: ResolvedVc<ModuleGraph>,
+    module: ResolvedVc<Box<dyn EcmascriptChunkPlaceable>>,
+    export_name: RcStr,
+) -> Result<Vc<bool>> {
+    let export_usage_info = compute_export_usage_info(graph)
+        .resolve_strongly_consistent()
+        .await?;
+
+    let export_usage_info = export_usage_info.await?;
+    let Some(exports) = export_usage_info.used_exports.get(&module) else {
+        return Ok(Vc::cell(true));
```

Comment:
can you add a comment for why we return `true` here.  if the `module` doesn't exist in the graph, what does that mean?